docs/_assets/attributes.adoc

```diff
@@ -21,25 +21,32 @@
 :toclevels: 2
 :icons: font
 
-//Latest version
-:KroxyliciousVersion: 0.13
-:gitRef: tree/main
+// Latest version
+:KroxyliciousVersion: 0.14.0-SNAPSHOT
+// Used for links to release artifacts, will be 404 during development
+:KroxyliciousReleaseTag: v{KroxyliciousVersion}
+// Used to point at source code and blobs, will point at main during development and at the release tag when deployed
+:KroxyliciousGitRef: main
```

Comment:
I wonder if things would be simpler if:

1. we deleted `KroxyliciousGitRef` and used `KroxyliciousReleaseTag` everywhere
2. In the release script, instead of updating the KroxyliciousVersion to 0.15.0-SNAPSHOT we could update it to 0.15.0, and at release time we wouldn't have to edit it in the release commit.

This would mean that the documentation would contain the hypothetical links that would exist after deployment. So in the snapshot website you would see links to the 0.15.0 release etc. Source code tree/blob and raw links wouldn't work either, but they would be in their final form.

With the way it is, pointing the ref at `main` for development you can at least have working links to tree, blob, raw at development time, maybe that is worth managing this extra variable.

Reply:
Git ref should mean we can build and release from a branch as well. I.e. build a 0.13.1 by branching from the `v0.13.0` tag and back porting some fixes.

Reply:
wouldn't we also be bumping KroxyliciousVersion to 0.13.1 and KroxyliciousReleaseTag to v0.13.1 in that case? 

What's different about the backport workflow?

Reply:
When you put it like that nothing.